modules/tinymce/src/themes/silver/test/ts/browser/editor/header/PromotionTest.ts

```diff
@@ -0,0 +1,64 @@
+import { ApproxStructure, Assertions, UiFinder } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SelectorFind } from '@ephox/sugar';
+import { TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.header.PromotionTest', () => {
+  const promotionClass = 'tox-promotion';
```

Comment:
I guess this could be a separate `promotionSelector` instead then it could be.
```const promotionSelector = '.tox-promotion';```

So then you wouldn't need the ```'.' + promotionClass``` two times. But since the `tox-promotion` is also in the approx structure you might want to keep the promotionClass and use that there but maybe that just makes the whole thing harder to read. DRY is fine for the most part but when it comes to test code I tend to like having a bit of duplication just so that you don't need to jump around just to understand a test case.